hub/package-manager/winget/pinning.md

```diff
@@ -0,0 +1,234 @@
+| **--name**   |  Limits the search to the name of the application. |  
+| **--moniker**   | Limits the search to the moniker listed for the application. |  
+| **--tag**   | Limits the search to the tag listed for the application. | 
+| **--cmd, --command**   | Limits the search to the command of the application. |   
+| **-s, --source**   |  Restricts the search to the source name provided. Must be followed by the source name. |  
+| **-e, --exact**   |   Uses the exact string in the query, including checking for case-sensitivity. It will not use the default behavior of a substring. |  
+| **--installed** | Pin a specific installed version |
+| **--accept-source-agreements** | Used to accept the source license agreement, and avoid the prompt. |
+| **--header** | Optional Windows-Package-Manager REST source HTTP header. |
+| **--wait** | Prompts the user to press any key before exiting |
+| **-o, --log**  |  Directs the logging to a log file. You must provide a path to a file that you have the write rights to. |
+| **--verbose-logs** | Used to override the logging setting and create a verbose log. |
+| **--disable-interactivity** | Disable interactive prompts. |
+
+### Examples
+
+The following example removes a pin for an application. 
+
+```CMD
+winget pin remove powertoys
+```
+
+The following example removes a pin for an application using its ID.
+
+```CMD
+winget pin remove --id Microsoft.PowerToys
+```
+
+## list
+
+The **list** subcommand lists all current pins, or full details of a specific pin. 
+
+Usage:
+
+```cmd
+winget pin list [[-q] <query>] [<options>]
+```
+
+### Arguments
+
+| Argument      | Description |
+|-------------|-------------|  
+| **-q,--query**  |  The query used to search for an app. |
+| **-?, --help** |  Get additional help on this command. |
+
+### Options
+
+The options allow you to customize listing pins to meet your needs.
+
+| Option  | Description |
+|-------------|-------------|  
+| **--id**    |  Limits the search to the ID of the application.   |  
+| **--name**   |  Limits the search to the name of the application. |  
+| **--moniker**   | Limits the search to the moniker listed for the application. |  
+| **--tag**   | Limits the search to the tag listed for the application. | 
+| **--cmd, --command**   | Limits the search to the command of the application. |   
+| **-s, --source**   |  Restricts the search to the source name provided. Must be followed by the source name. |  
+| **-e, --exact**   |   Uses the exact string in the query, including checking for case-sensitivity. It will not use the default behavior of a substring. |  
+| **--accept-source-agreements** | Used to accept the source license agreement, and avoid the prompt. |
+| **--header** | Optional Windows-Package-Manager REST source HTTP header. |
+| **--wait** | Prompts the user to press any key before exiting |
+| **-o, --log**  |  Directs the logging to a log file. You must provide a path to a file that you have the write rights to. |
```

Comment:
```suggestion
| **--logs, --open-logs**  |  Open the default logs location |
```
Same as above comment. This doesn't seem to be supported on 1.5 rc and 1.6 preview. Instead I see --logs, --open-logs